src/components/TokenHolderList/TokenHolderListItem.tsx

```diff
@@ -10,21 +10,26 @@ import GalleryLink from '~/components/core/GalleryLink/GalleryLink';
 import { BaseXL } from '~/components/core/Text/Text';
 import { useMemberListPageActions } from '~/contexts/memberListPage/MemberListPageContext';
 import { TokenHolderListItemFragment$key } from '~/generated/TokenHolderListItemFragment.graphql';
+import { TokenHolderListItemQueryFragment$key } from '~/generated/TokenHolderListItemQueryFragment.graphql';
 import useDebounce from '~/hooks/useDebounce';
 import { useBreakpoint } from '~/hooks/useWindowSize';
 import detectMobileDevice from '~/utils/detectMobileDevice';
 import { removeNullValues } from '~/utils/removeNullValues';
 import { graphqlTruncateUniversalUsername } from '~/utils/wallet';
 
+import HoverCardOnUsername from '../HoverCard/HoverCardOnUsername';
 import MemberListGalleryPreview from './TokenHolderListGalleryPreview';
 
 type Props = {
   tokenHolderRef: TokenHolderListItemFragment$key;
   direction: Directions.LEFT | Directions.RIGHT;
   fadeUsernames: boolean;
+  queryRef: TokenHolderListItemQueryFragment$key;
 };
 
-function TokenHolderListItem({ tokenHolderRef, direction, fadeUsernames }: Props) {
+const DISABLED_GALLERY_PREVIEW = true;
```

Comment:
Added a flag to disabled the 3 images preview on community page. Just in case, if we wanna reuse the code again